google/cloud/storage/examples/run_examples_utils.sh

```diff
@@ -122,48 +124,16 @@ run_all_bucket_examples() {
 
   # The list of commands in the storage_bucket_samples program that we will
   # test. Currently get-metadata assumes that $bucket_name is already created.
-  readonly BUCKET_EXAMPLES_COMMANDS="list-buckets get-bucket-metadata list-objects"
-
-  if [ ! -x storage_bucket_samples ]; then
-    echo "${COLOR_YELLOW}[  SKIPPED ]${COLOR_RESET}" \
-        " storage_bucket_samples is not compiled"
-    return
-  fi
-  local object_name="object-$(date +%s)"
-  for example in ${BUCKET_EXAMPLES_COMMANDS}; do
-    log="$(mktemp --tmpdir "storage_bucket_samples_${example}.XXXXXXXXXX.log")"
-    echo    "${COLOR_GREEN}[ RUN      ]${COLOR_RESET}" \
-        "storage_bucket_samples ${example} running"
-    ./storage_bucket_samples \
-        ${example} "${bucket_name}" >"${log}" 2>&1 </dev/null
-    if [ $? = 0 ]; then
-      echo  "${COLOR_GREEN}[       OK ]${COLOR_RESET} ${example}"
-      continue
-    else
-      echo    "${COLOR_RED}[    ERROR ]${COLOR_RESET} ${example}"
-      echo
-      echo "================ [begin ${log}] ================"
-      cat "${log}"
-      echo "================ [end ${log}] ================"
-    fi
-    /bin/rm -f "${log}"
-  done
+  readonly BUCKET_EXAMPLES_COMMANDS=$(tr '\n' ',' <<_EOF_
```

Comment:
[no action needed]

That's a neat, readable way to join a list of strings that haven't seen before in Bash :)